packages/create-svelte/templates/default/src/error.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```

Comment:
I kinda feel like we should remove these files from the templates, because they make it seem like this (rather than `src/routes/+error.svelte`) is the proper place to define your error page, when in reality this is the absolute last resort, and a framework default is totally acceptable (if we just make it a bit nicer than it currently is). Thoughts?

Reply:
Sounds sensible. It could be a little overwhelming with all these files adding up.